tlair/src/main/scala/at/forsyte/apalache/tla/lir/formulas/Base.scala

```diff
@@ -0,0 +1,26 @@
+package at.forsyte.apalache.tla.lir.formulas
+
+import at.forsyte.apalache.tla.lir.formulas.StandardSorts.FunctionSort
+
+abstract class Sort(val sortName: String)
+trait HasSort {
+  val sort: Sort
+}
```

Comment:
We might want to, in the future, externalize function definitions (to de-hybridize `FunDef`)  or declarations. Those aren't terms, but they'd still have associated sorts. For now though, I guess it makes more sense to just move this into Term, since pulling it out if we need to later is trivial.